controls/V-233511.rb

```diff
@@ -60,15 +60,18 @@
     its('output') { should eq input('pg_port') }
   end
 
-  if virtualization.system == 'docker'
-    describe 'The docker container must have networking tools to check its ports and their processes' do
-      skip 'If the currently defined port configuration is deemed prohibited, this is a finding.'
-    end
-
-  else
-    describe port(input('pg_port')) do
-      it { should be_listening }
-      its('processes') { should match ['postmaster'] }
-    end
+  if !input('aws_rds')
```

Comment:
Making conditions based on the aws_rds input may be a good use of the `only_if` syntax that we are putting more of in RHEL8.

For example:
```
  only_if('Control not applicable - SSH is not installed within containerized RHEL', impact: 0.0) {
    !virtualization.system.eql?('docker') || file('/etc/ssh/sshd_config').exist?
  }
```

or in this context something like below (the syntax may not be perfect, but the idea is there):

```
  only_if('Control not applicable - in AWS RDS', impact: 0.0) {
    !input(aws_rds)
  }
```

Reply:
@em-c-rod Is the 'Control not applicable - SSH is not installed within containerized RHEL' just a comment in the code, or does it actually show up for the user looking at the test results? If the later, we need to be concise on the language/context of this statement.

Reply:
![image](https://github.com/mitre/crunchy-data-postgresql-stig-baseline/assets/34140975/14f59283-5168-4a34-9c41-507c1edf3fc1)
Can we improve the cosmetics?
![image](https://github.com/mitre/crunchy-data-postgresql-stig-baseline/assets/34140975/c1623fa8-3a45-4d28-8cf3-e1c893a97a40)